docs/howto-install-learn-ocaml-client.md

```diff
@@ -0,0 +1,24 @@
+How to install the learn-ocaml client
+=====================================
+
+This section explains how to install a learn-ocaml client for students.
+
+## Manual compilation
+
+Note: you need a working ```opam``` environment with OCaml ```4.05.0```.
+
+* Install the dependencies using:
+  - if you have opam >=2.0 available:
+``
+opam switch create . --deps-only && opam install opam-installer && eval $(opam env)
+``
+  - otherwise:
+``
+make build-deps
+``
+(You may want to first read the script `scripts/install-opam-deps.sh` to check what it does.)
+
+* Compile the learn-ocaml management tool using:
+```
+make && make opaminstall
+```
```

Comment:
In this case, it's probably simpler to just remove the `--deps-only` option above, so this whole section could be replaced by just:
```
opam switch create . --locked && eval $(opam env)
```

no need to call `make` manually ! :)

Reply:
Note that this applies not just to the client -- the others probably need updating.
And it's also possible to install without even cloning the source manually, e.g.:
```
opam switch create learn-ocaml 4.05.0 # optional if the user is already on a 4.05.0 switch
opam pin https://github.com/ocaml-sf/learn-ocaml.git --locked
eval $(opam env)
```